GetDataType.js

```diff
@@ -1,19 +1,18 @@
 const data = require('./response.json');
+const dataTypes = [];
 function loadData(e) {
-
-      
-      console.log(data);
-      const deff = data.library.statements.def;
-      const filtered = deff.filter(defined);
-      for (let i = 0; i < filtered.length; i++) {
-        console.log(filtered[i].expression.operand.dataType);
-      }
-      function defined(set) {
-          for (let j = 0; j < deff.length; j++) {
-            if (set.expression.operand !== undefined) return set;
-          }
-      }
-    
+  const deff = data.library.statements.def;
+  const filtered = deff.filter(defined);
+  for (let i = 0; i < filtered.length; i++) {
+    let dataType=filtered[i].expression.operand.dataType;
+    dataTypes[i]=dataType;
```

Comment:
Small comment, but you can just use `.push` here instead of setting the element in the array at index `i`. E.g.

``` JavaScript
dataTypes.push(filtered[i].expression.whatever);
```
```